test_trello_endtoend.py

```diff
@@ -31,6 +33,8 @@ def driver():
 
 @pytest.fixture(scope='module')
 def test_app():
+    load_dotenv()
```

Comment:
After you make the other changes I suggest, you will want to set override=True in order to be able to run all of your tests together, irrespective of order. So if the integration tests loaded .env.test, this test should override it again with the real .env values.